app/src/main/java/com/kickstarter/ui/adapters/projectcampaign/ViewElementAdapter.kt

```diff
@@ -24,18 +27,16 @@ import com.kickstarter.ui.viewholders.projectcampaign.ExternalViewViewHolder
 import com.kickstarter.ui.viewholders.projectcampaign.ImageElementViewHolder
 import com.kickstarter.ui.viewholders.projectcampaign.TextElementViewHolder
 import com.kickstarter.ui.viewholders.projectcampaign.VideoElementViewHolder
-import com.trello.rxlifecycle.FragmentEvent
-import rx.Observable
+import io.reactivex.subjects.BehaviorSubject
 
 /**
  * Adapter Specific to hold a list of ViewElements from the HTML Parser
  */
 class ViewElementAdapter(
     val requireActivity: FragmentActivity,
     private val fullScreenDelegate: FullScreenDelegate,
-    private val lifecycle: Observable<FragmentEvent>
-) : RecyclerView
-.Adapter<RecyclerView.ViewHolder>() {
+    private val lifecycleObservable: BehaviorSubject<KSLifecycleEvent> = BehaviorSubject.create()
+) : RecyclerView.Adapter<RecyclerView.ViewHolder>(), DefaultLifecycleObserver {
```

Comment:
https://developer.android.com/topic/libraries/architecture/lifecycle.html 
ProjectCampaignFragment is the LifecycleOwner
ViewElementAdapter is the LifecycleObserver and overrides callback functions onPause() and onDestroy()